### PR TITLE
EVG-7705: Add triaging functionality to expanded metrics in legacy UI

### PR DESCRIPTION
**Pull request by zamj**

### CODE REVIEW

File: public/static/app/common/ApiUtil.js [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
+    },
+    httpPoster: function (base) {
+      let realBase = base || "";
+      realBase = realBase.endsWith("/")
+        ? realBase.substr(0, realBase.length - 1)
+        : realBase;
+
+      return function (apiEndpoint, endpointTplParams, body) {
+        if (apiEndpoint[0] !== "/") apiEndpoint = "/" + apiEndpoint;
+        data = JSON.stringify(body);
```
Comment (john-m-liu) [inline public/static/app/common/ApiUtil.js:28]: I believe $http accepts an object in the data param so you don't need to do this

File: public/static/app/common/ApiUtil.js [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+        return $http({
+          method: "POST",
+          url: realBase + _.template(apiEndpoint)(endpointTplParams),
+          withCredentials: true,
+          data: data,
+          headers: {
+            "Content-Type": "text/plain",
```
Comment (john-m-liu) [inline public/static/app/common/ApiUtil.js:35]: Shouldn't you let angular figure out the content-type?

File: public/static/app/common/constants.js [hunk 3]
```diff
@@ -144,7 +144,8 @@ mciModule
   .constant('CANARY_EXCLUSION_REGEX', /^((?!canary_|fio_|iperf|NetworkBandwidth).*)$/)
   .constant('CEDAR_APP_URL', 'https://cedar.mongodb.com')
   .constant("PERFORMANCE_ANALYSIS_AND_TRIAGE_API", {
-    BASE:'https://signal-processing-service.server-tig.prod.corp.mongodb.com',
+    BASE:'https://signal-processing-service.server-tig.staging.corp.mongodb.com',
```
Comment (john-m-liu) [inline public/static/app/common/constants.js:147]: Was this change intentional? This is what will be in production evergreen

File: public/static/app/common/ApiUtil.js (hunk 1 above)
Comment (zamj) [reply public/static/app/common/ApiUtil.js:28]: Because evergreen is hosted outside of the ELB for Kanopy, every request we send has to have our corp secure cookie on it. As well, these requests have to deal with CORS because it's a different origin. When you do anything but a simple request (gets, posts with a content-type of text/plain), a preflight OPTIONS request is sent. `$http` doesn't send the corp secure cookie on these preflight requests so the ELB redirects them to login and the overall request fails.

By setting the content-type to text/plain explicitly and stringifying the body, we avoid these preflight requests and the cookie gets set correctly.

The other option here is to work with the Kanopy team and see about not requiring cookies on OPTION requests as technically they're not required. That's being tracked [here](https://jira.mongodb.org/browse/DEVOPS-1812). This lets us get around the issue in the meantime.

File: public/static/app/common/ApiUtil.js (hunk 2 above)
Comment (zamj) [reply public/static/app/common/ApiUtil.js:35]: See [here](https://github.com/evergreen-ci/evergreen/pull/3663#discussion_r437452408)

File: public/static/app/common/constants.js (hunk 3 above)
Comment (zamj) [reply public/static/app/common/constants.js]: Whoops, fixed
